packages/ui/components/ui/tx/index.tsx

```diff
@@ -4,7 +4,7 @@ import { ActionViewComponent } from './action-view';
 import { ViewBox, ViewSection } from './viewbox';
 import { getStakingTokenMetaData } from './registry';
 import { ValueView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1/asset_pb';
-import { ValueViewComponent } from './value';
+import { ValueViewComponent } from '../value';
 
 export const TransactionViewComponent = ({ txv }: { txv: TransactionView }) => {
```

Comment:
`TransactionViewComponent` is the only transaction-related component that was used by both Minifront and Prax inside the monstrous `tx` directory. I extracted other components and cleaned the directory up a little